src/cmd/build.rs

```diff
@@ -731,26 +771,22 @@ mod tests_ci_only {
 
                 // we choose zero optimization passes as the "cli" parameter
                 optimization_passes: Some(OptimizationPasses::Zero),
+                keep_debug_symbols: false,
             };
 
-            // when
```

Comment:
Why did you remove the `when` and `then`? The `given` is still there and [I think that it's generally a good structure for tests](https://martinfowler.com/bliki/GivenWhenThen.html).

Reply:
I think it got lost somehow. I went back and forth on this part until I had it working. I will restore it.